App/src/Twig/FormatNumber.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class FormatNumber extends AbstractExtension
+{
+    public function getFunctions()
+    {
+        return array(
+            new TwigFunction('formatNumber', array($this, 'addDigits')),
+        );
+    }
+
+    public function addDigits(int $amount)
```

Comment:
The name of the function itself was fine 🆗 

```suggestion
            new TwigFunction('formatNumber', array($this, 'addDigits')),
        );
    }

    public function formatNumber(int $amount)
```

Reply:
Je fais cette modification,  il faut que je remplace le addDigits également dans la array ($this .... ) , seulement, si je fais ca, tout sera nommé 'formatNumber' , est-ce que c'est une bonne idée du coup ?